pulser-core/pulser/json/abstract_repr/backend.py

```diff
@@ -18,6 +18,7 @@
     _convert_complex,
     _deserialize_noise_model,
 )
+from pulser.json.exceptions import AbstractReprError
```

Comment:
This will trigger a `DeprecationWarning` once you merge develop. The new location is `pulser.exceptions.serialization`